boards/posix/native_posix/cmdline_common.c

```diff
@@ -10,9 +10,9 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <math.h>
-#include "posix_trace.h"
-#include "posix_board_if.h"
-#include "zephyr/types.h"
+#include <zephyr.h>
```

Comment:
and here

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/20119#discussion_r341810118

Reply:
Did you forget this one or is it needed? (https://github.com/zephyrproject-rtos/zephyr/pull/20119#discussion_r341839246)